front/migrations/20241211_parents_migrator.ts

```diff
@@ -190,7 +190,61 @@ const migrators: Record<ConnectorProvider, ProviderMigrator | null> = {
   },
   google_drive: null,
   microsoft: null,
-  github: null,
+  github: {
+    transformer: (nodeId, parents) => {
+      const repoId = parents[parents.length - 1];
+      // case where we are already good
+      if (repoId.startsWith("github-repository")) {
+        return { parents, parentId: parents[1] };
+      }
+      if (parents.length < 3) {
```

Comment:
are we sure? e.g. there's no github element that's purely at the root, or with just one direct parent?
likely true but important to check

with folders there are single & two parents cases

Reply:
I'm gonna grep on this warn anyway

Reply:
I had forgotten the return btw